tests/api/test_security.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import unittest
+import os
+
+
+class Testsecurity(object):
+    def setup(self):
+        # app['TESTING'] = True
+        # self.app = app.test_client()
+        pass
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
```

Comment:
these tests have to make sure that the user used is really in the database so maybe create and delete them before and after the test. But this might be implemented when we implement the the config file